src/table/src/TableRow.js

```diff
@@ -54,11 +54,21 @@ export default class TableRow extends PureComponent {
     }
   }
 
+  handleRef = node => {
+    this.node = node
+  }
+
   handleKeyPress = e => {
     if (this.props.isSelectable) {
       if (e.key === 'Enter' || e.key === ' ') {
         this.props.onSelect()
-        e.preventDefault()
+
+        // NOTE: In scenarios where the target for key press is our
+        // tabIndex <Pane> below, prevent the default event action
+        // so that browser scroll will not occur.
+        if (e.target.parentNode === this.node) {
```

Comment:
Because of the wrapping `div` below, we compare the event target's parent node to our ref node here. If they match, the event was fired while keyboard focus was on the `tabIndex` `<Pane>` below and we should stop browser scroll by preventing the default event action.

If the event target was some other node, then the user has interacted with other types of nested UI, and we should *not* prevent the default action so that `input` and `change` events can fire normally